.github/workflows/CI.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+on: [ push, pull_request ]
+jobs:
+  Build:
+
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        node-version: [ 14 ]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
```

Comment:
Any reason we use v1 and not v2 considering we are now starting with GitHub Actions (which is the latest) (Sorry if we already decided something here, I know that @nikku and @pinussilvestrus discussed it already, but I dont know the outcome)

Reply:
No particular reason. Happy to take your PR on our default workflow templates that up the version there: 
https://github.com/bpmn-io/.github/tree/master/workflow-templates.